docs/content/routing/entrypoints.md

```diff
@@ -737,3 +737,59 @@ entrypoints.websecure.http.tls.domains[1].sans=foo.test.com,bar.test.com
     entrypoints.websecure.address=:443
     entrypoints.websecure.http.tls.certResolver=leresolver
     ```
+
+### listen on some ip addresses only
```

Comment:
```suggestion
### Listen on a single IP address only
```

Reply:
This is true for one entry point. The example shows several entry points with different ip addresses. And the examples for the orchestrators show several as well.